packages/api-clients/open-api/notificationConfigApi.yml

```diff
@@ -95,6 +101,103 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/Problem"
+        "404":
+          description: Tenant notification config not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /internal/tenantNotificationConfigs:
```

Comment:
For creation routes, I used the `/internal` prefix which I saw on some existing routes. I send the tenant id (and user id, for user configs) in the body, which, comparing to other creation routes, seems more consistent than putting it as a path parameter